test/boostTest.cpp

```diff
@@ -148,6 +148,9 @@ void initializeOptions()
 }
 }
 
+// TODO: Prototype -- why is this needed?
+test_suite* init_unit_test_suite( int /*argc*/, char* /*argv*/[] );
```

Comment:
Isn't there a boost header for this prototype?

Reply:
Not sure... We shouldn't be using it in general... https://www.boost.org/doc/libs/1_74_0/libs/test/doc/html/boost_test/adv_scenarios/obsolete_init_func.html

I think there's some issue about it somewhere...

Reply:
To use a global fixture (https://www.boost.org/doc/libs/1_74_0/libs/test/doc/html/boost_test/tests_organization/fixtures/global.html) ? Sounds nicer. Can you create an issue?